tests/unit/testplan/report/test_testing.py

```diff
@@ -209,6 +253,16 @@ def test_merge(self):
         assert rep.logs == rep2.logs
         assert rep.entries == rep2.entries
 
+    def test_hash(self):
+        """Test that a consistent hash can be generated for a report object."""
+        rep_1 = TestCaseReport(name="testcase1")
+        rep_2 = TestCaseReport(name="testcase2")
+
+        for rep in rep_1, rep_2:
+            assert rep.hash == rep.hash
```

Comment:
Basically just checking the same hash is returned every time for the same object. Since hash is a `@property`, each of these is actually a separate function call under the hood, so technically it could return a different value every time (which would be really stupid).